### ISSUE TITLE
LDM generate full of Noise

### ISSUE THREAD

**Issue by TienMinhHoa** [gaozhihan/PreDiff#40]
I use your model and train on my own dataset. My Dataset contains about 4000 samples for training. I do not know whether it is enough for training or not. How many samples do you have for training? The VAE can reconstruct the image in a perfect way. But the LDM cannot generate a meaningfull image.
This is an example in training set at epoch 80. 
<img width="1800" height="900" alt="Image" src="https://github.com/user-attachments/assets/2a832de8-7fe0-46f9-9292-7cea3d4f2e89" />

What happen with me ?

**Comment by gaozhihan** [comment]
Thank you for your interest in our work. You may find it helpful to try the following steps to identify and address the errors:

1. **Disable the adversarial loss** by [setting its weight](https://github.com/gaozhihan/PreDiff/blob/30ce3a056fbc310adb41f4a936af4e98f9951277/scripts/vae/sevirlr/cfg.yaml#L76) to 0.

2. **Adjust the learning rate** — in most cases, using a smaller [lr](https://github.com/gaozhihan/PreDiff/blob/30ce3a056fbc310adb41f4a936af4e98f9951277/scripts/vae/sevirlr/cfg.yaml#L29) helps alleviate this issue.
